ui/components/Collapsible.tsx

```diff
@@ -1,47 +1,46 @@
 import { Collapse } from "@material-ui/core";
 import React from "react";
+import styled from "styled-components";
+import Flex from "./Flex";
 import Icon, { IconType } from "./Icon";
+import Text from "./Text";
 
 const Collapsible = ({ children }) => {
```

Comment:
can we avoid the inline styles in this component?

Reply:
What concerns -as i don't really get it here :eyes:-  ?